model/testutil/api.go

```diff
@@ -55,7 +55,7 @@ func SetupAPITestData(testConfig *evergreen.Settings, taskDisplayName string, va
 	// Unmarshal the project configuration into a struct
 	project := &model.Project{}
 	ctx := context.Background()
-	opts := model.GetProjectOpts{
```

Comment:
I think it's awkward that we can have opts but not have ReadFileFrom set (I know this is a dummy case but). If opts != nil in LoadProjectInto, should we default ReadFileFrom? Or error log if it's empty (and add something to this case?)